extensions/linear/src/api/getTeams.ts

```diff
@@ -16,29 +17,32 @@ export type TeamResult = Pick<
 } & {
   activeCycle?: Pick<Cycle, "id" | "number">;
 } & {
-  projects: { nodes: Project["id"][] };
+  membership?: {
+    sortOrder: number;
+  };
 };
 
 export async function getTeams() {
-  const { graphQLClient } = getLinearClient();
+  const { graphQLClient, linearClient } = getLinearClient();
+
+  const me = await linearClient.viewer;
+
   const { data } = await graphQLClient.rawRequest<{ teams: { nodes: TeamResult[] } }, Record<string, unknown>>(
     `
-      query {
+      query($userId: String!) {
         teams {
           nodes {
             id
             name
             icon
             color
-            projects {
```

Comment:
Wasn't used anywhere, plus it's not really cheap request.

Reply:
ty ty for the cleanup 🫶